pages/common/flyctl.md

```diff
@@ -0,0 +1,36 @@
+# flyctl
+
+> Command-line tool for flyctl.io.
+> More information: <https://github.com/superfly/flyctl>.
+
+- Sign into fly account:
+
+`flyctl auth login`
+
+- Launch from your own Dockerfile (Default path is current working directory):
+
+`flyctl launch --dockerfile {{path/to/Dockerfile}}`
+
+- Open browser to current deployed application:
```

Comment:
```suggestion
- Open the browser to the current deployed application:
```
If it opens the application in the current e.g. directory → `current, deployed`
If there's only one running application, which you open → `currently deployed`

Reply:
@einverne which one of those cases does apply to the example?

Reply:
Perhaps this better
```suggestion
- Open the current deployed application in the default web browser:
```

Reply:
It is, but doesn't really solve the issue.

It's not clear whether it opens the `currently deployed application` or the `deployed application in the current directory`